rfc8446.xml

```diff
@@ -4523,14 +4523,18 @@ The registry [shall be/ has been] initially populated with the values described
 <xref target="signature-algorithms"/>. The following values SHALL be marked as
 “Recommended”: ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384,
 rsa_pss_rsae_sha256, rsa_pss_rsae_sha384, rsa_pss_rsae_sha512,
-rsa_pss_pss_sha256, rsa_pss_pss_sha384, rsa_pss_pss_sha512, and ed25519.</t>
+rsa_pss_pss_sha256, rsa_pss_pss_sha384, rsa_pss_pss_sha512, and ed25519. IETF
+standards-track RFCs that assign new values MUST specify the value to use for
+the Recommended column (other allocations get value 'N').</t>
```

Comment:
I don't think that this is right.  I think that you want to say: "The Recommended column is assigned a value of "N" unless explicitly requested, and only IETF standards-track RFCs can request a value of "Y"."  That is, rather than require a value for the column, imply a default value.

This text implies that a standards-track RFC can't choose to set the value to "N".

Reply:
I'm not sure it implies that, but I do like being explicit about the "N" being the default.